esp-hal/src/soc/esp32/peripherals.rs

```diff
@@ -38,8 +38,8 @@ crate::peripherals! {
     HINF <= HINF,
     I2C0 <= I2C0,
     I2C1 <= I2C1,
-    I2S0 <= I2S0 (I2S0),
-    I2S1 <= I2S1 (I2S1),
+    [I2s0] I2S0 <= I2S0 (I2S0),
```

Comment:
Could you please elaborate what does this mean?

Reply:
Added a doc comment for the peripherals macro